iterators.tex

```diff
@@ -6442,56 +6461,84 @@
 container copies the elements, which cannot be done in constant time.
 \exitexample
 
-\begin{codeblock}
+\begin{itemdecl}
 @\newtxt{template <class T>}@
-@\newtxt{struct is_view;}@
+@\newtxt{struct enable_view \{ \};}@
 
 @\newtxt{struct view_base \{ \};}@
 
+@\newtxt{template <class T>}@
+@\newtxt{constexpr bool \xname{view_predicate}; // \expos}@
```

Comment:
[dcl.constexpr]/1:

> The `constexpr` specifier shall be applied only to the definition of a variable or variable template, the
declaration of a function or function template, or the declaration of a static data member of a literal
type (3.9).

So no, it is not valid.